ONSdigital/ssdc-rm-notify-service
# 2921 Add configurable throttle to sms request enriched receiver

# Motivation and Context
We do not watch to hit the gov notify rate limit during action rule runs. A crude, short term, solution to reduce the rate we're sending requests at in case we find ourselves hitting that limit is a configurable sleep in the message receiver.

# What has changed
- Add configurable sleep to SMS request enriched receiver
- Remove unused configuration variables

# How to test?
Configure a non-zero delay in the service config and run an SMS action rule, the rate of requests to notify should be reduced.

# Links
https://trello.com/c/K2L4McyN/2921-add-a-configurable-throttle-to-notify-service-to-control-rate-of-sms-sending-crudely
